sshkey_paramiko.r2py

```diff
@@ -94,9 +94,6 @@ class _sshkey_paramiko_BER(object):
   def __init__(self, content=''):
       self.content = content
       self.idx = 0
-
-  def decode(self):
```

Comment:
Thanks for the comment. I added a simple explanation to where the decode function was. Is this okay?